calibration_details:
  image_width_px: 1232
  image_height_px: 1072
  timestamp: '2025-04-29 16:32:13'
  threshold_value: 50
  poly_approx_epsilon: 0.01
  min_contour_area: 10.0
nozzles:
- index: 0
  spray_pattern_pixels:
  - - 88
    - 962
  - - 120
    - 963
  - - 153
    - 984
  - - 165
    - 978
  - - 174
    - 954
  - - 201
    - 944
  - - 205
    - 928
  - - 214
    - 938
  - - 222
    - 929
  - - 202
    - 907
  - - 200
    - 924
  - - 167
    - 922
  - - 140
    - 901
  - - 120
    - 914
  - - 115
    - 936
- index: 1
  spray_pattern_pixels:
  - - 130
    - 900
  - - 130
    - 950
  - - 200
    - 950
  - - 200
    - 900
- index: 2
  spray_pattern_pixels:
  - - 170
    - 900
  - - 170
    - 950
  - - 230
    - 950
  - - 230
    - 900
- index: 3
  spray_pattern_pixels:
  - - 204
    - 926
  - - 240
    - 949
  - - 252
    - 951
  - - 263
    - 947
  - - 272
    - 932
  - - 298
    - 919
  - - 305
    - 909
  - - 293
    - 895
  - - 267
    - 896
  - - 261
    - 882
  - - 239
    - 880
  - - 211
    - 904
  - - 213
    - 913
- index: 4
  spray_pattern_pixels:
  - - 246
    - 931
  - - 286
    - 934
  - - 311
    - 952
  - - 329
    - 922
  - - 373
    - 901
  - - 367
    - 894
  - - 322
    - 893
  - - 308
    - 881
  - - 314
    - 880
  - - 298
    - 875
  - - 298
    - 882
  - - 281
    - 883
  - - 256
    - 912
  - - 273
    - 907
- index: 5
  spray_pattern_pixels:
  - - 338
    - 873
  - - 347
    - 889
  - - 319
    - 893
  - - 316
    - 907
  - - 290
    - 934
  - - 320
    - 943
  - - 338
    - 958
  - - 360
    - 947
  - - 366
    - 961
  - - 364
    - 939
  - - 374
    - 935
  - - 367
    - 931
  - - 394
    - 913
- index: 6
  spray_pattern_pixels:
  - - 324
    - 932
  - - 331
    - 940
  - - 353
    - 942
  - - 382
    - 958
  - - 404
    - 950
  - - 431
    - 919
  - - 423
    - 904
  - - 411
    - 905
  - - 353
    - 884
  - - 358
    - 890
  - - 352
    - 886
  - - 344
    - 893
  - - 349
    - 901
  - - 342
    - 904
  - - 343
    - 920
- index: 7
  spray_pattern_pixels:
  - - 430
    - 871
  - - 407
    - 896
  - - 406
    - 906
  - - 414
    - 908
  - - 400
    - 916
  - - 407
    - 924
  - - 379
    - 943
  - - 389
    - 951
  - - 417
    - 947
  - - 416
    - 956
  - - 425
    - 953
  - - 425
    - 961
  - - 442
    - 967
  - - 454
    - 962
  - - 459
    - 939
  - - 496
    - 915
  - - 491
    - 906
  - - 460
    - 902
  - - 452
    - 893
  - - 446
    - 898
  - - 451
    - 887
  - - 435
    - 886
- index: 8
  spray_pattern_pixels:
  - - 543
    - 914
  - - 522
    - 908
  - - 517
    - 898
  - - 493
    - 905
  - - 474
    - 891
  - - 441
    - 929
  - - 424
    - 932
  - - 421
    - 953
  - - 461
    - 950
  - - 466
    - 959
  - - 490
    - 967
  - - 505
    - 940
  - - 540
    - 924
- index: 9
  spray_pattern_pixels:
  - - 585
    - 898
  - - 547
    - 898
  - - 535
    - 887
  - - 546
    - 885
  - - 516
    - 878
  - - 493
    - 898
  - - 492
    - 911
  - - 469
    - 930
  - - 470
    - 937
  - - 499
    - 939
  - - 520
    - 959
  - - 539
    - 950
  - - 549
    - 927
  - - 584
    - 911
- index: 10
  spray_pattern_pixels:
  - - 628
    - 894
  - - 615
    - 882
  - - 600
    - 886
  - - 587
    - 880
  - - 589
    - 875
  - - 563
    - 869
  - - 565
    - 862
  - - 550
    - 872
  - - 541
    - 867
  - - 504
    - 930
  - - 515
    - 936
  - - 540
    - 933
  - - 551
    - 945
  - - 573
    - 949
  - - 601
    - 915
  - - 625
    - 910
- index: 11
  spray_pattern_pixels:
  - - 555
    - 917
  - - 587
    - 917
  - - 617
    - 937
  - - 622
    - 928
  - - 630
    - 932
  - - 639
    - 907
  - - 643
    - 912
  - - 668
    - 894
  - - 675
    - 881
  - - 627
    - 874
  - - 595
    - 859
  - - 599
    - 865
  - - 579
    - 893
  - - 573
    - 889
- index: 12
  spray_pattern_pixels:
  - - 596
    - 915
  - - 623
    - 932
  - - 622
    - 924
  - - 631
    - 922
  - - 638
    - 931
  - - 665
    - 935
  - - 680
    - 930
  - - 683
    - 911
  - - 722
    - 899
  - - 727
    - 888
  - - 714
    - 877
  - - 673
    - 882
  - - 636
    - 864
  - - 618
    - 880
  - - 628
    - 883
  - - 622
    - 891
  - - 615
    - 888
  - - 619
    - 900
  - - 606
    - 902
- index: 13
  spray_pattern_pixels:
  - - 642
    - 939
  - - 660
    - 953
  - - 671
    - 952
  - - 670
    - 961
  - - 680
    - 960
  - - 682
    - 968
  - - 691
    - 963
  - - 699
    - 969
  - - 715
    - 962
  - - 717
    - 950
  - - 758
    - 927
  - - 763
    - 913
  - - 725
    - 904
  - - 718
    - 908
  - - 703
    - 894
  - - 683
    - 893
  - - 670
    - 917
  - - 647
    - 928
- index: 14
  spray_pattern_pixels:
  - - 723
    - 865
  - - 700
    - 896
  - - 674
    - 910
  - - 667
    - 923
  - - 673
    - 933
  - - 708
    - 934
  - - 723
    - 948
  - - 743
    - 950
  - - 756
    - 941
  - - 761
    - 926
  - - 776
    - 921
  - - 791
    - 894
  - - 770
    - 877
  - - 758
    - 880
  - - 741
    - 868
- index: 15
  spray_pattern_pixels:
  - - 828
    - 911
  - - 821
    - 903
  - - 801
    - 902
  - - 797
    - 888
  - - 768
    - 884
  - - 759
    - 889
  - - 766
    - 891
  - - 759
    - 893
  - - 749
    - 917
  - - 719
    - 937
  - - 724
    - 945
  - - 763
    - 951
  - - 754
    - 946
  - - 761
    - 944
  - - 791
    - 959
  - - 805
    - 944
  - - 807
    - 929
- index: 16
  spray_pattern_pixels:
  - - 767
    - 923
  - - 779
    - 930
  - - 809
    - 926
  - - 819
    - 935
  - - 813
    - 938
  - - 837
    - 939
  - - 855
    - 912
  - - 885
    - 903
  - - 897
    - 881
  - - 854
    - 878
  - - 849
    - 883
  - - 830
    - 870
  - - 814
    - 870
  - - 800
    - 898
  - - 776
    - 910
- index: 17
  spray_pattern_pixels:
  - - 928
    - 870
  - - 904
    - 866
  - - 887
    - 874
  - - 871
    - 860
  - - 850
    - 863
  - - 841
    - 876
  - - 843
    - 885
  - - 819
    - 917
  - - 826
    - 926
  - - 844
    - 923
  - - 875
    - 939
  - - 895
    - 924
  - - 896
    - 912
  - - 931
    - 893
- index: 18
  spray_pattern_pixels:
  - - 980
    - 889
  - - 966
    - 881
  - - 952
    - 882
  - - 946
    - 875
  - - 905
    - 870
  - - 888
    - 879
  - - 888
    - 899
  - - 866
    - 922
  - - 898
    - 928
  - - 919
    - 945
  - - 935
    - 935
  - - 945
    - 911
  - - 979
    - 899
- index: 19
  spray_pattern_pixels:
  - - 1013
    - 906
  - - 1001
    - 895
  - - 984
    - 897
  - - 966
    - 886
  - - 965
    - 873
  - - 931
    - 901
  - - 921
    - 920
  - - 903
    - 929
  - - 899
    - 940
  - - 937
    - 947
  - - 927
    - 955
  - - 970
    - 958
  - - 997
    - 943
  - - 986
    - 935
  - - 1012
    - 914
- index: 20
  spray_pattern_pixels:
  - - 1055
    - 894
  - - 1042
    - 887
  - - 1019
    - 891
  - - 998
    - 875
  - - 984
    - 875
  - - 969
    - 884
  - - 972
    - 896
  - - 961
    - 906
  - - 933
    - 917
  - - 929
    - 932
  - - 957
    - 941
  - - 965
    - 934
  - - 967
    - 940
  - - 976
    - 937
  - - 986
    - 946
  - - 1001
    - 947
- index: 21
  spray_pattern_pixels:
  - - 1038
    - 847
  - - 1011
    - 865
  - - 1003
    - 887
  - - 998
    - 888
  - - 999
    - 872
  - - 995
    - 889
  - - 979
    - 908
  - - 988
    - 922
  - - 1019
    - 923
  - - 1028
    - 933
  - - 1053
    - 937
  - - 1096
    - 881
- index: 22
  spray_pattern_pixels:
  - - 1137
    - 898
  - - 1129
    - 890
  - - 1107
    - 889
  - - 1090
    - 875
  - - 1076
    - 882
  - - 1078
    - 872
  - - 1040
    - 911
  - - 1032
    - 911
  - - 1021
    - 930
  - - 1033
    - 939
  - - 1054
    - 938
  - - 1079
    - 950
  - - 1100
    - 947
- index: 23
  spray_pattern_pixels:
  - - 1183
    - 903
  - - 1124
    - 882
  - - 1113
    - 885
  - - 1099
    - 907
  - - 1056
    - 937
  - - 1061
    - 941
  - - 1056
    - 941
  - - 1060
    - 947
  - - 1093
    - 940
  - - 1113
    - 952
  - - 1141
    - 950
  - - 1154
    - 926
  - - 1170
    - 921
- index: 24
  spray_pattern_pixels:
  - - 1231
    - 890
  - - 1196
    - 890
  - - 1192
    - 879
  - - 1181
    - 873
  - - 1161
    - 874
  - - 1165
    - 877
  - - 1151
    - 881
  - - 1147
    - 895
  - - 1134
    - 896
  - - 1135
    - 903
  - - 1119
    - 922
  - - 1130
    - 930
  - - 1153
    - 927
  - - 1171
    - 941
  - - 1185
    - 942
  - - 1200
    - 920
  - - 1231
    - 908
